std-bits/base/src/main/java/org/enso/base/time/Date_Period_Utils.java

```diff
@@ -0,0 +1,25 @@
+package org.enso.base.time;
+
+import java.time.temporal.ChronoField;
+import java.time.temporal.Temporal;
+import java.time.temporal.TemporalAdjusters;
+
+public class Date_Period_Utils implements TimeUtilsBase {
```

Comment:
We could do this as:
```
public class FirstDayOfQuarter implements TemporalAdjuster {
  @Override
  public Temporal adjustInto(Temporal temporal) {
    int currentQuarter = YearMonth.from(temporal).get(IsoFields.QUARTER_OF_YEAR);
    return temporal.with(IsoFields.QUARTER_OF_YEAR, currentQuarter).with(TemporalAdjusters.firstDayOfMonth());
  }
}
```

and just have the single adjust flow?

Reply:
That's a splendid idea! I will try to implement it.